code-studio/src/js/assessments/multi.js

```diff
@@ -1,11 +1,9 @@
 /* global $, Dialog, appOptions, CDOSounds */
 
-var Multi = function (id, levelId, standalone, numAnswers, answers, lastAttemptString) {
+var Multi = window.Multi = function (id, standalone, numAnswers, answers, lastAttemptString) {
```

Comment:
Another pattern we've been trying to follow is to adapt files like this to follow AMD and export their class, then have the entry file for the module actually load things onto window, so that we are touching global scope in as few places as possible.  See all the stuff loaded onto window.dashboard in code-studio.js.